### ISSUE TITLE
Any way to completely revert?

### ISSUE THREAD

**Issue by modelorona** [CasperVerswijvelt/Better-Internet-Tiles#52]
Hi there, thanks for the app. I do find it very useful coming from a Samsung.

I do have a curiosity - is there a way to completely reset? I searched this repo, xda, and whatever forums I could find, but wasn't able to find a concrete answer.

I was playing around with the app to see what would happen when I archived it and I was unable to see the original Internet tile on my Pixel 7a. Reinstalling returned it but I noticed it was the BIT variant.

**Comment by CasperVerswijvelt** [comment]
AFAIK removing the tiles from your QS and uninstalling is the only way, not sure what causes this behaviour...

**Comment by modelorona** [comment]
I'll give it a try. In all my attempts, I left the tiles on top as is

**Comment by modelorona** [comment]
Nope, still the same behaviour. Removed all the custom QS tiles, uninstalled, restarted phone, stock Internet tile is still missing.

I don't think this is a problem for me going forward as I prefer it split, but it could become one down the line.

Let me know if you'd like me to send over any logs.

**Comment by zetcco** [comment]
Yup, this happened for me too. Original Internet tile is missing.

**Comment by modelorona** [comment]
@zetcco I ended up bringing back the original combined Internet tile by force closing BIT, removing its shell access, downloading SystemUI Tuner (https://apt.izzysoft.de/fdroid/index/apk/com.zacharee1.systemuituner), granting it shell access, and then toggling off the combined internet tile and then back on.

**Comment by zetcco** [comment]
Would you be able to tell me how did you toggled on/off the combined internet tile on the System UI Tuner? I long pressed on it, removed it with red remove icon, then added back in. But cannot see any change :(

**Comment by zetcco** [comment]
Thank you! @modelorona It worked.

**Comment by beeshyams** [comment]
@modelorona thanks but unfortunately this method doesn't work on Android 16, Pixel 8

@CasperVerswijvelt thanks for your work. Any solution for this. I don't want to use Shizuku any more and restore the original tiles. Can you please look into this? 

Edit: Uninstalling BIT and Shizuku restored stock tile after a reboot
